lib/dom.js

```diff
@@ -88,6 +88,28 @@ declare class Event {
     BUBBLING_PHASE: number;
 }
 
+declare class UIEvent extends Event {
+    detail: number;
+}
+
+declare class MouseEvent extends UIEvent {
+    altKey: boolean;
+    button: number;
+    buttons: number;
+    clientX: number;
+    clientY: number;
+    ctrlKey: boolean;
+    metaKey: boolean;
+    movementX: number;
+    movementY: number;
+    region: number;
```

Comment:
thanks, fixed in https://github.com/facebook/flow/commit/00f6411eb721cc46ef89bdc8cc0274e15ead82b1